formula-android/src/test/java/com/instacart/formula/fragment/FragmentFlowStoreTest.kt

```diff
@@ -55,15 +55,15 @@ class FragmentFlowStoreTest {
         val detail = Detail(1)
         store
             .state(FragmentEnvironment())
-            .map { it.states }
+            .map { it.states.mapKeys { it.key.key } }
```

Comment:
`key.key` reads weirdly

Reply:
Agreed, I've updated it to `entry.key.key` though it only makes it slightly better.